undertow/src/main/java/org/wildfly/extension/undertow/DeploymentDefinition.java

```diff
@@ -102,13 +102,12 @@ protected void executeRuntimeStep(OperationContext context, ModelNode operation)
                 return;
             }
 
-            final UndertowDeploymentService deploymentService = (UndertowDeploymentService) controller.getService();
             SessionStat stat = SessionStat.getStat(operation.require(ModelDescriptionConstants.NAME).asString());
-
             if (stat == null) {
                 context.getFailureDescription().set(UndertowLogger.ROOT_LOGGER.unknownMetric(operation.require(ModelDescriptionConstants.NAME).asString()));
             } else {
                 ModelNode result = new ModelNode();
+                UndertowDeploymentService deploymentService = (UndertowDeploymentService) controller.getService();
```

Comment:
this is odd, as it shouldn't do any difference if this is here or there.
This service must be present and up in any way as we check that earlier.
I don't mind the change, but don't see how could this fix the problem.